src/sections/pools/pool/details/PoolDetails.tsx

```diff
@@ -46,14 +46,31 @@ export const PoolDetails = () => {
   const meta = pool.meta
   const omnipoolFee = useOmnipoolFee()
 
+  const isTransferModalOpen =
+    isOpen && (pool.meta.isStableSwap || pool.meta.isErc20)
+  const isGeth = !ixXYKPool && pool.isGETH
+
+  const isGethAssetPreselected = isTransferModalOpen && isGeth
```

Comment:
rename it to isGETHPool pls, since you do not select any assets in asset selector in this case.